src/patternfly/components/OverflowMenu/examples/OverflowMenu.md

```diff
@@ -276,6 +277,5 @@ The action group consists of a primary and secondary action. Any additional acti
 | `.pf-c-overflow-menu__control` | `<div>` | Initiates the overflow menu control. **Required** |
 | `.pf-c-overflow-menu__group` | `<div>` | Initiates an overflow menu group. |
 | `.pf-c-overflow-menu__item` | `<div>` | Initiates an overflow menu item. **Required** |
-| `.pf-c-overflow-menu__separator` | `<li>` | Defines a separator within the menu. |
```

Comment:
Since this is a promoted component, we want to keep the documentation for the old separator around - not only here but in examples, so that we don't inadvertently change something that breaks the existing `__separator` until it's removed and no longer supported. We have a breaking change issue to update components' implementation of their own separators with the divider component that this work would be great to included in - https://github.com/patternfly/patternfly-next/issues/2621